languagetool-language-modules/zh/src/main/java/org/languagetool/language/SimplifiedChinese.java

```diff
@@ -0,0 +1,15 @@
+package org.languagetool.language;
```

Comment:
Please add our standard license header here as in other *.java files (but with your name). Same for all other new *.java files.